README.md

```diff
@@ -30,6 +30,12 @@ PGPORT=5432
 # for Postgres versions < 9.4
 PGPASS=password
 ```
+# Email G00dness
+
+EMAIL=hlr.dev.useyouremailN00Bulous@gmail.com
+EMAILSERVICE=Gmail (note the cap)
```

Comment:
Will this work with anything but Gmail? Can I send emails straight from my development machine w/o a third-party?